tests/integration/framework/process/daprd/daprd.go

```diff
@@ -327,17 +328,41 @@ func (d *Daprd) ProfilePort() int {
 	return d.profilePort
 }
 
+type MetricBucket struct {
+	UpperBound      float64
+	CumulativeCount uint64
+}
+
+type metricsOptions struct {
+	includeBuckets bool
+}
```

Comment:
Please move `includeBuckets` as a field to `options` in `options.go` with an appropriate name.

Reply:
removed